build-support/bin/generate_github_workflows.py

```diff
@@ -75,7 +75,7 @@ def hashFiles(path: str) -> str:
 PYTHON37_VERSION = "3.7"
 PYTHON38_VERSION = "3.8"
 PYTHON39_VERSION = "3.9"
-ALL_PYTHON_VERSIONS = [PYTHON37_VERSION, PYTHON38_VERSION, PYTHON39_VERSION]
+ALL_PYTHON_VERSIONS = [PYTHON39_VERSION]
```

Comment:
Maybe worth de-generalizing this code so that it's simpler? Sounds like we never expect to support more than 1 version at a time.